Assets/Code/Scripts/Bullet.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using UnityEngine;
+
+namespace MJ.Weapon
+{
+    public class Bullet : MonoBehaviour
+    {
+        private int _weaponID; // 무기 아이디랑 같음
+        private float _attackDamage; // 무기 데미지
+        private int _piercing; // 관통 횟수
+        private float _bulletSpeed; // 총알 속도
+        private Rigidbody2D _rigidbody;
+        public void Init(int weaponID, float attackDamage, int piercing, float bulletSpeed, Vector3 direction)
```

Comment:
변수와 함수 간에 단락 구분 부탁드립니다 + private public 간 또는 유사한 변수 묶음들을 구분지어서 단락구분을 지어주는 것도 좋습니다

```suggestion

        public void Init(int weaponID, float attackDamage, int piercing, float bulletSpeed, Vector3 direction)
```